strapi/strapi
# fix: add type compare in row isSelected condition

### What does it do?

Add `type` comparing in `isSelected` condition checks in `TableRows`

### Why is it needed?

Current condition checks only `id` of row element but there is a case (happens in side  a new folder) the first folder and the first asset have the same `id`

<img width="1057" alt="Screenshot at May 22 22-51-22" src="https://github.com/GitStartHQ/client-strapi/assets/51897872/f8b6b9b9-5cf3-4d30-ab1a-bed8bed58bdf">

### How to test it?

Create a new folder in `MediaLibrary`, inside that new folder, create a new folder and upload a new asset. Verify that it works as expected.

### Related issue(s)/PR(s)

Closes https://github.com/strapi/strapi/issues/16377
